### ISSUE TITLE
Unhandled exception

### ISSUE THREAD

**Issue by VMBindraban** [SkytAsul/BeautyQuests#12]
**Describe the bug**
Running `/quests` throws an error but does open the menu

**To Reproduce**

Run `/quests`

**Screenshots, errors**
```
org.bukkit.command.CommandException: Unhandled exception executing command 'quests' in plugin BeautyQuests v0.1
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot-1.12.2.jar:git-Spigot-2cf50f0-8166d17]
at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[spigot-1.12.2.jar:git-Spigot-2cf50f0-8166d17]
at org.bukkit.craftbukkit.v1_12_R1.CraftServer.dispatchCommand(CraftServer.java:648) ~[spigot-1.12.2.jar:git-Spigot-2cf50f0-8166d17]
at net.minecraft.server.v1_12_R1.PlayerConnection.handleCommand(PlayerConnection.java:1397) [spigot-1.12.2.jar:git-Spigot-2cf50f0-8166d17]
at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1232) [spigot-1.12.2.jar:git-Spigot-2cf50f0-8166d17]
at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:45) [spigot-1.12.2.jar:git-Spigot-2cf50f0-8166d17]
at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:1) [spigot-1.12.2.jar:git-Spigot-2cf50f0-8166d17]
at net.minecraft.server.v1_12_R1.PlayerConnectionUtils$1.run(SourceFile:13) [spigot-1.12.2.jar:git-Spigot-2cf50f0-8166d17]
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:514) [?:?]
at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46) [spigot-1.12.2.jar:git-Spigot-2cf50f0-8166d17]
at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:748) [spigot-1.12.2.jar:git-Spigot-2cf50f0-8166d17]
at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:406) [spigot-1.12.2.jar:git-Spigot-2cf50f0-8166d17]
at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:679) [spigot-1.12.2.jar:git-Spigot-2cf50f0-8166d17]
at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:577) [spigot-1.12.2.jar:git-Spigot-2cf50f0-8166d17]
at java.lang.Thread.run(Thread.java:844) [?:?]
Caused by: java.lang.NoClassDefFoundError: javafx/util/Pair
at fr.skytasul.quests.gui.Inventories.put(Inventories.java:175) ~[?:?]
at fr.skytasul.quests.gui.Inventories.create(Inventories.java:46) ~[?:?]
at fr.skytasul.quests.QuestsCommand.onCommand(QuestsCommand.java:45) ~[?:?]
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot-1.12.2.jar:git-Spigot-2cf50f0-8166d17]
... 15 more
```

**Softwares:**
ViaVersion 1.5.1
Spigot 1.12.2
BeautyQuests 0.10

**Additional context**

`/quests create` throws the same error and does nothing.

**Comment by SkytAsul** [comment]
Yes it's caused by a missing Java library. Someone already had this problem, he posted his fix on my Discord server. Otherwise, this problem will be fixed for all in the next BeautyQuests update !

**Comment by SkytAsul** [comment]
Fixed on 0.12
